.github/workflows/scripts/index-versions.sh

```diff
@@ -48,11 +50,19 @@ create_pkg_in_channel() {
     else
         # We need to recreate channel-fuel-latest.toml, generating new URLs and sha256 hashes for the download links.
         printf "%s: new version available: %s -> %s\n" "${1}" "${3}" "${2}"
+        printf "%s: Generating new package\n" "${1}"
         create_new_pkg "$1" "$2"
+        _header="$RETVAL"
+        _content=""
         for target in "${_targets[@]}"; do
-            printf "[pkg.%s.target.%s]\n" "$1" "${target}" >>channel-fuel-latest.toml
+            _content+="[pkg.${1}.target.${target}]\n"
             add_url_and_hash $_repo "$2" "$_tarball_prefix-${target}.tar.gz"
+            _content+="$RETVAL"
         done
+
+        # Only write to file if there's no problem downloading and hashing all the above releases.
+	_package=$(printf "%s\n%s" "${_header}" "${_content}")
+	echo -ne "$_package" >>channel-fuel-latest.toml
```

Comment:
This indentation looks odd in the diff, but when looking at the raw file it [seems fine](https://github.com/FuelLabs/fuelup/blob/ec2d282f1f7ad8a1a6bdaa0fe8c26ec402ed9dbd/.github/workflows/scripts/index-versions.sh#L63-L65). :ghost: Just going to trust our bash fmt checker :)

Reply:
This uses tabs instead of spaces. Should the formatter enforce that only spaces are used @bingcicle?

Reply:
Manually ran `shfmt -i 4 -ci` for this particular file, I think I will open up another PR to enable `shellcheck` and `shfmt` for the `.github/workflows/scripts` directory since it's likely we add more scripts

Reply:
Nice one github, two different of rendering tab indentation on the same site :+1: :100: